shared/components/customEmojis.tsx

```diff
@@ -0,0 +1,17 @@
+import { s } from "../styles";
+import styled from "styled-components";
+
+export const EmojiImage = styled.img`
+  width: 24px;
```

Comment:
The display size should probably be 16x16, there is a 4px space around all emojis.